packages/traceloop-sdk/tests/experiment/test_experiment.py

```
@@ -105,3 +116,253 @@ def test_parse_jsonl_to_rows_complex_json_objects(experiment):
+                slug="relevance-checker",
+                required_input_fields=["prompt", "response"],
+            ),
+        ]
+
+        # Mock dataset response
+        experiment._datasets.get_version_jsonl = Mock(
+            return_value='{"columns":{}}\n{"input": "test"}'
+        )
+
+        # Run with stop_on_error=True - should stop after error
+        results, errors = await experiment._run_locally(
```

Ruff (0.14.7)

Unpacked variable `results` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)